runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -119,15 +121,38 @@
 	private static final Pattern NUMBER_PATTERN = Pattern.compile("([0-9]+)");
 	private static final Pattern LEVEL_UP_PATTERN = Pattern.compile(".*Your ([a-zA-Z]+) (?:level is|are)? now (\\d+)\\.");
 	private static final Pattern BOSSKILL_MESSAGE_PATTERN = Pattern.compile("Your (.+) kill count is: <col=ff0000>(\\d+)</col>.");
+
+	private static final ImmutableList<Pattern> PVP_KILL_MESSAGES = ImmutableList.of(
```

Comment:
The type should remain a `List` imo.

```suggestion
	private static final List<Pattern> PVP_KILL_MESSAGES = ImmutableList.of(
```

Reply:
The other statement is wrong then